src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,17 @@
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users.json';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+      || null;
+}
+
+export const TodoInfo = ( {todo} ) => (
+  <article className={`TodoInfo ${todo.completed === true ? 'TodoInfo--completed' : ''}`}>
```

Comment:
```suggestion
  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
```
it is better to use libraries like classnames or clsx for conditional classNames